pages/common/starship.md

```diff
@@ -0,0 +1,36 @@
+# starship
+
+> The minimal, blazing-fast, and infinitely customizable prompt for any shell.
+> More information: <https://starship.rs>.
+
+- Print the shell function used to execute starship:
```

Comment:
I like the following:
```suggestion
- Generate starship integration code for the current shell and print it to stdout:
```

Reply:
Actually it doesn't generate nothing, only prints the selected script.

Reply:
But I don't think that's a necessary detail—usually "generate" refers to this type of thing (if if currently is it just a dump of a static file's contents).

Reply:
To me `generate` also sounds like it is creating the code when you execute the command. How about:
```suggestion
- Print starship integration code for the specified shell:
```

Reply:
I think I like @marchersimon's best, but with a slightly grammatical tweak:
```suggestion
- Print the starship integration code for the specified shell:
```